src/routers/v3/RouteRouter.js

```diff
@@ -45,15 +44,12 @@ class RouteRouter extends ApplicationRouter<Object> {
         destinationName,
         end: routes[0].endCoords,
         originName,
-        routeId: routes[0].routeId,
```

Comment:
do they not need routeId/routeName/whatever it is? I thought this was being converted to the GTFS routeId

Reply:
no this wasn't a value in the first place -- a route id corresponds to each direction, but this is an entire route which doesnt have a route id -- i know, confusing

Reply:
if you read my description you will notice that this is our own routeId, which is from our analytics code -- has nothing to do with an actual GTFS routeId, which corresponds to a direction